Fog.js

```diff
@@ -1017,8 +1017,18 @@ function redraw_light_walls(clear=true){
 		}
 		let drawnWall = new Boundary(new Vector(x/adjustedScale/window.CURRENT_SCENE_DATA.scale_factor, y/adjustedScale/window.CURRENT_SCENE_DATA.scale_factor), new Vector(width/adjustedScale/window.CURRENT_SCENE_DATA.scale_factor, height/adjustedScale/window.CURRENT_SCENE_DATA.scale_factor))
 		window.walls.push(drawnWall);
-
 	}
+	let darknessfilter = (window.CURRENT_SCENE_DATA.darkness_filter != undefined) ? window.CURRENT_SCENE_DATA.darkness_filter : 0;
+ 	if(darknessfilter == 0 && window.walls.length>4){
+ 		$('#light_container').css('mix-blend-mode', 'unset');

```

Comment:
nit: I think you can stack these so the browser doesn't try rendering each time.

`$('#light_container').css({'mix-blend-mode': 'unset', 'background': '#0000',  'opacity': '0.3'});

Reply:
this is true, didn't really think about that rerendering. I've used both ways of doing it throughout a lot of the code as have other people. It might be worth going through and adjust this in other places too.

I'll look into it if/when I've got the time. 

Thanks for the tip.